Source/WebCore/platform/gamepad/cocoa/GameControllerGamepad.mm

```diff
@@ -149,6 +157,18 @@ static void disableDefaultSystemAction(GCControllerButtonInput *button)
     };
 }
 
+void GameControllerGamepad::playEffect(GamepadHapticEffectType, const GamepadEffectParameters&, CompletionHandler<void(bool)>&& completionHandler)
+{
+    // FIXME: Implement.
```

Comment:
Better to use FIXME(<bugzilla>) so we know a bug exists to do the work.

Also consider using 'notImplemented()' here.

Reply:
Ok.